### DISCUSSION TITLE
[Enhancement] Improved SEO for MAUI

### DISCUSSION THREAD

**Discussion by luiseduardohd** [dotnet/maui#1664] [Ideas]
## Summary
Te current word Maui could present some troubles when developers try to search for some information, the word is currently meaning an island, a semigod, a Disney character, this can drag some troubles to find technical information about the technology, i have seen this problem in other technologies and frameworks, even mobile frameworks, like Titanium ( metal ), Corona ( beer ), and as you know, these mobile Frameworks are already dead, I'm not saying that the technology was not a factor, but the problem with the name could have taken a role in this, even .net had this problem at the very beginning, because some search engines didn't take the dot or they gave it special meaning and the word net alone was used in different context at that time. React does not seems to be affected by this issue because the name is a verb not a noun, and Unity has not other meaning on tech, so they are number one on search results, but i suspect that we can have some trouble to get first results with MAUI. With Xamarin i have never had this problem before, because it was a new word and it didn't conflict even with the Tamarin animal.

## API Changes

The change should be just a renaming of the namespace to the desired one or no change depending on the decision.

Please consider changing the name to something more SEO friendly , currently i don't have an specific idea or word in mind, developers seems to like the word maui so we can try something simple like .MAUI ( .net MAUI ), MSMAUI (Microsoft Multi-platform App UI ), MAUIX ( Microsoft Multi-platform App UI/UX ),  XMAUI ( XamarinMAUI ), i don't know , we can propose a brainstorm, or maybe someone from Marketing on Microsoft can give some ideas.

## Intended Use Case

When some developer wants to search something with this name, the developer should be able to find the desired information on first result.

**Comment by GeraudFabien** [comment]
Related to and maybe duplicate has in a nutshell you want to change the name.
https://github.com/dotnet/maui/issues/34
https://github.com/dotnet/maui/issues/35
https://github.com/dotnet/maui/discussions/41
